src/components/banner-image/banner-image.js

```diff
@@ -1,31 +1,44 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-export const BannerImage = ({ image: {src, alt, title}, link }) => {
-  return (
-    <div>
-      {link ?
-        <a href={link} target="_blank" rel="noopener noreferrer">
-          <img src={src} alt={alt} title={title}/>
-        </a>
-        :
-        <img src={src} alt={alt} title={title}/>
-      }
-    </div>
+const handleClick = ({ path, shouldOpenLinkOnNewPage }) => {
+  const windowInstance = window.open(
+    path,
+    shouldOpenLinkOnNewPage ? "_blank" : "_self"
   );
+  windowInstance.focus();
 };
 
+export const BannerImage = ({
+  image: { src, alt, title, width, height },
+  link
+}) => (
+  <div>
+    <img
+      src={src}
+      alt={alt}
+      title={title}
+      {...(!!link && { onClick: () => handleClick(link) })}
+      {...(!!width && { width })}
+      {...(!!height && { height })}
+    />
+  </div>
+);
+
 BannerImage.propTypes = {
   image: PropTypes.shape({
     src: PropTypes.string.isRequired,
     alt: PropTypes.string,
-    title: PropTypes.string
+    title: PropTypes.string,
+    width: PropTypes.string,
```

Comment:
Aren't width and height numbers ?

Reply:
Width and height can be either `number` or `string`. 

For example, you can set the value of width to `100`, `100px` or `50%`.

Reply:
Width and height attributes for `<img />` tag only supports integer. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img

Reply:
@Utwo Just updated this, thank you!